pkg/server/test/write_authzmodel.go

```diff
@@ -17,6 +17,11 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func conditionedRelation(rel *openfgav1.RelationReference, condition string) *openfgav1.RelationReference {
```

Comment:
Mind adding this to the Typesystem package instead? 

See https://github.com/openfga/openfga/blob/main/pkg/typesystem/typesystem.go#L62-L144 for similar helpers.